src/Symfony/Component/ExpressionLanguage/Node/GetAttrNode.php

```diff
@@ -149,4 +149,12 @@ public function toArray()
                 return [$this->nodes['node'], '[', $this->nodes['attribute'], ']'];
         }
     }
+
+    public function __unserialize(array $data): void
+    {
+        $this->nodes = $data['nodes'];
```

Comment:
@nicolas-grekas I'm wondering if there's a way to just call the "default PHP unserialization" in one line and then add the extra checks?

Reply:
use __wakeup instead?

Reply:
on a closer look, the code is correct (except for isNullCoalesce which has never been part of any serialized payloads)